CloudAppSecurityDocs/access-policy-aad.md

```diff
@@ -37,19 +37,19 @@ To create a new access policy, follow this procedure:
 
     - **Device tags**: Use this filter to identify unmanaged devices.
 
-    - **Location**: Use this filter to identify unknown (and therefore risky) locations.
+    - **Location**: This filter identifies unknown (and therefore risky) locations.
```

Comment:
This change makes the language in the list inconsistent. Consider leaving as-is or changing lines 38 and 42 to follow the "This filter..." construction as well.